client/components/map/FacilityCard.tsx

```diff
@@ -1,49 +1,208 @@
-import { Flex, Text, HStack, Button } from "@chakra-ui/react";
-import { TStateFacilities } from "app-context/SheetyContext/types";
+import { Flex, Text, HStack, Button, Box, VStack, Divider } from "@chakra-ui/react";
+import { TEmptyItem, TItemSelection, TStateFacilities } from "app-context/SheetyContext/types";
 import { Image } from "@chakra-ui/react";
+import { ChevronDownIcon, ChevronUpIcon, ExternalLinkIcon } from "@chakra-ui/icons";
 import { COLORS } from "theme";
+import React, { ReactNode, useState } from "react";
+import styles from "components/home/hideScrollbar.module.css";
+import Link from "next/link";
+import { useSheetyData } from "hooks/useSheetyData";
+import { CheckIcon } from "@chakra-ui/icons";
+import { Categories } from "api/sheety/enums";
 export const FacilityCard = ({
+	items,
 	facCardDetails,
 	facCardDistance,
-	width,
-	left,
 }: {
+	items: (TItemSelection | TEmptyItem)[];
 	facCardDetails: TStateFacilities;
 	facCardDistance: number;
-	width: string;
-	left: string;
 }) => {
+	const { getItemCategory } = useSheetyData();
+
+	const TRANSLATION_DIST = -50;
+	const [translateY, setTranslateY] = useState(0);
+	const handleMovement = () => {
+		translateY === TRANSLATION_DIST ? setTranslateY(0) : setTranslateY(TRANSLATION_DIST);
+	};
+	const widths = ["86%", "50%", "40%", "25%"];

```

Comment:
This looks a bit odd at the second break-point. Not sure if that's expected. 

The following is from an iPad emulation on FIrefox.

![Screen Shot 2023-12-01 at 17 36 44](https://github.com/bettersg/recyclegowhere/assets/5859290/6d992f57-f9a9-4020-ab51-c24dba2f299c)